deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -294,7 +305,7 @@ def default_generator(self,
           pad_batches=pad_batches):
 
         feed_dict = dict()
-        if y_b is not None and not predict:
+        if y_b is not None:
```

Comment:
Removing this breaks predict_on_batch.  I'm trying to figure out a good way to solve it though in my PR here -- https://github.com/deepchem/deepchem/pull/1041

Reply:
That will be great, I didn't realize this could be destructive.